KubeArmor/config/config.go

```diff
@@ -129,7 +129,7 @@ func readCmdLineParams() {
 	seLinuxProfileDirStr := flag.String(ConfigSELinuxProfileDir, "/tmp/kubearmor.selinux", "SELinux profile directory")
 	criSocket := flag.String(ConfigCRISocket, "", "path to CRI socket (format: unix:///path/to/file.sock)")
 
-	visStr := flag.String(ConfigVisibility, "process,file,network,capabilities", "Container Visibility to use [process,file,network,capabilities,none]")
+	visStr := flag.String(ConfigVisibility, "process,network,capabilities", "Container Visibility to use [process,file,network,capabilities,none]")
```

Comment:
There is no need to change this, since this will be overridden by the value passed from the configmap.